subgraphs/uniswap-v3/schema.graphql

```diff
@@ -791,6 +797,50 @@ type Swap implements Event @entity {
   pool: LiquidityPool!
 }
 
+type Collect implements Event @entity {
```

Comment:
Should have we used `Withdrawal` for these events? Since they are like a withdrawal of accrued fees from the liquidity pool. If anything we can add an `isCollect` field on the withdrawal entity or something. The rest of fields seem to be pretty much the same in both entities.

If we count fee collections separately we'll have to handle them downstream everywhere. I think sticking to the standard dex schema, when it makes sense to, is valuable. What do you think?

Reply:
Yeah - I think this makes sense and is a good idea!